src/components/overrides/LastUpdated.astro

```diff
@@ -0,0 +1,30 @@
+---
+import { REPO, getCommitInfo } from "../../utils/git";
+
+const { lang, lastUpdated } = Astro.locals.starlightRoute;
+const filePath = Astro.locals.starlightRoute.entry.filePath;
+
+const info = await getCommitInfo(filePath);
+---
+
+{
+  lastUpdated && (
+    <p>
+      Last updated:
+      <time datetime={lastUpdated.toISOString()}>
+        {lastUpdated.toLocaleDateString(lang, {
+          dateStyle: "medium",
+          timeZone: "UTC",
+        })}
+      </time>
+      {info && (
+        <>
+          {" by "}
+          <a href={info.committer.href}>{info.committer.name}</a>
+          {" in "}
+          <a href={`https://github.com/${REPO}/commit/${info.hash}`}>{info.hash.substring(0, 7)}</a>
```

Comment:
Maybe better get the two hashs and not make a substring? (With two hashes i mean the large and short)